src/sentry/rules/match.py

```diff
@@ -0,0 +1,40 @@
+from collections import OrderedDict
+
+
+class MatchType:
+    CONTAINS = "co"
+    ENDS_WITH = "ew"
+    EQUAL = "eq"
+    GREATER_OR_EQUAL = "gte"
+    IS_SET = "is"
+    LESS_OR_EQUAL = "lte"
+    NOT_CONTAINS = "nc"
+    NOT_ENDS_WITH = "new"
+    NOT_EQUAL = "ne"
+    NOT_SET = "ns"
+    NOT_STARTS_WITH = "nsw"
+    STARTS_WITH = "sw"
+
+
+LEVEL_MATCH_CHOICES = OrderedDict(
+    [
+        (MatchType.EQUAL, "equal to"),
+        (MatchType.GREATER_OR_EQUAL, "greater than or equal to"),
+        (MatchType.LESS_OR_EQUAL, "less than or equal to"),
+    ]
+)
```

Comment:
Is this one being used anywhere? i don't think I see it imported in this diff, not even on `level.py`

Reply:
Good catch that's a bug.